test/functional/pdo_sqlsrv/pdo_nested_query_mars.phpt

```diff
@@ -1,82 +1,70 @@
 --TEST--
 fetch multiple result sets with MARS on and then off
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-
-include 'MsCommon.inc';
+require_once("MsCommon_mid-refactor.inc");
 
 function NestedQuery_Mars($on)
 {
-    require("MsSetup.inc");
-    
-    $tableName = GetTempTableName();
-          
-    $conn = new PDO( "sqlsrv:server=$server;database=$databaseName;MultipleActiveResultSets=$on", $uid, $pwd);
-    $conn->SetAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
-    
-    $stmt = $conn->exec("CREATE TABLE $tableName ([c1_int] int, [c2_varchar] varchar(20))");
-    
-    $query = "INSERT INTO $tableName ([c1_int], [c2_varchar]) VALUES (1, 'Dummy value 1')";
-    $stmt = $conn->query($query);
-    
-    $query = "INSERT INTO $tableName ([c1_int], [c2_varchar]) VALUES (2, 'Dummy value 2')";
-    $stmt = $conn->query($query);
+    $tableName = getTableName();
+
+    $conn = connect("MultipleActiveResultSets=$on");
+
+    createTable($conn, $tableName, array("c1_int" => "int", "c2_varchar" => "varchar(20)"));
 
-    $query = "SELECT * FROM $tableName ORDER BY [c1_int]";
+    insertRow($conn, $tableName, array("c1_int" => 1, "c2_varchar" => "Dummy value 1"));
+    insertRow($conn, $tableName, array("c1_int" => 2, "c2_varchar" => "Dummy value 2"));
+
```

Comment:
suggestion: make c1_int an identity column such that you can use `ODER BY`

Reply:
hmmm I don't want to change it so it doesn't change the test on the non AE side

Reply:
it's up to you, but this c1_int is basically an identity column ;)